### ISSUE TITLE
Missing NPC in Slave Pens Heroic. (BB #1155)

### ISSUE THREAD

**Issue by OregonAsari** [OregonCore/OregonCore#1155]
This issue was migrated from bitbucket.
**Original Reporter:** Selphius
**Original Date:** 24.11.2015 17:57:32 GMT+0000
**Original Priority:** minor
**Original Type:** bug
**Original State:** resolved
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1155

<hr>

There should be [Skar'this the Heretic](http://www.wowhead.com/npc=22421/skarthis-the-heretic), who give quest [The Cudgel of Kar'desh](http://www.wowhead.com/quest=13431/the-cudgel-of-kardesh) for Serpentshrine Attunement, even if there is no longer attu required, quest should be there as well as this NPC.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 24.11.2015 17:57:59 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1155/#comment-23555993

<hr>

This should solve the problem:

```
#!mysql

INSERT INTO `creature` (`guid`, `id`, `map`, `spawnMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `spawndist`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`) VALUES (1285586, 22421, 547, 2, 0, 0, -303.529, -467.7, 3.0355, 0.848033, 25, 0, 0, 14258, 0, 0);

```

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** kotishe
**Original Date:** 24.11.2015 18:28:41 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1155/#comment-23556698

<hr>

@Selphius maybe make pull requests you fixes in main repo?

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** Selphius
**Original Date:** 30.11.2015 14:42:28 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1155/#comment-23685433

<hr>

Can be pushed to repo. Resolved.

**Comment by OregonAsari** [comment]
This comment was migrated from bitbucket.
**Original User:** desteny
**Original Date:** 01.12.2015 23:59:48 GMT+0000
**Direct Link:** https://bitbucket.org/oregon/oregoncore/issues/1155/#comment-23730247

<hr>

Fix #1155 thanks @Selphius

→ <<cset 11ff2aec81fb>>
